game-cards/SGgame.js

```diff
@@ -300,6 +300,14 @@ exports.commands = {
 			if (!user.console) return;
 			return this.sendReply('Not Avaliable');
 		},
+		home: function (target, room, user) {
+			if (!user.console) return;
+			user.console.update(false, false, false);
```

Comment:
Theres an issue with this, suppose you were playing the game (the finished version) and you hit reset while on route. when you hit the back button you get the default green and white screen and not the route. To fix this ill shortly be adding `prevScreen` to the console, which tracks the last screen setup. So in preperation of that change this to.

```javascript
user.console.update(this.prevScreen[0], this.prevScreen[1], this.prevScreen[2]);
```